akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -31,5 +33,50 @@ static void reduceExample() throws InterruptedException, ExecutionException, Tim
         // 55
         //#reduce-operator-example
     }
-}
 
+    static void takeLastExample() throws InterruptedException, ExecutionException, TimeoutException {
+        //#takeLast-operator-example
+        // tuple of (Name, GPA)
+        List<Tuple2> sortedStudents = Arrays.asList(new Tuple2("Benita", 2.1), new Tuple2("Adrian", 3.1),
+                new Tuple2("Alexis", 4), new Tuple2("Kendra", 4.2), new Tuple2("Jerrie", 4.3), new Tuple2("Alison", 4.7));
```

Comment:
For the Java APIs and samples we use `akka.util.Pair` instead of the scala tuple type

Reply:
done.